src/io/flutter/view/FlutterView.java

```diff
@@ -89,6 +89,7 @@ public void dispose() {
   protected static final String INSTALLATION_TIMED_OUT_LABEL = "Waiting for JxBrowser installation timed out. Restart your IDE to try again.";
   protected static final String INSTALLING_DEVTOOLS_LABEL = "Installing DevTools...";
   protected static final String DEVTOOLS_FAILED_LABEL = "Setting up DevTools failed.";
+  protected static final int INSTALLATION_WAIT_LIMIT_SECONDS = 200;
```

Comment:
is there a reason to not make this limit 10X higher? Users may have limited bandwidth.

Reply:
I have no opinion on this and chose a random number. This doesn't end the download attempt, only the completable future, but since we display a message suggesting a restart we should probably just make this bigger.